lib/kafo_parsers/puppet_strings_module_parser.rb

```diff
@@ -130,6 +133,24 @@ def docs
 
     private
 
+    def self.run_puppet(command)
+      env_vars = self.puppet_bin.include?('/opt/puppetlabs') ? clean_env_vars : ::ENV
+      command = command.unshift(self.puppet_bin)
+      Open3.capture3(env_vars, *command, :unsetenv_others => true)
+    end
+
+    def self.clean_env_vars
+      # Cleaning ENV vars and keeping required vars only because,
+      # When using SCL it adds GEM_HOME and GEM_PATH ENV vars.
+      # These vars break foreman-maintain tool.
+      # This way we use system ruby to execute the bash script.
+      whitelisted_vars = %w[HOME USER LANG]
+
+      cleaned_env = ::ENV.select { |var| whitelisted_vars.include?(var) || var.start_with?('LC_') }
+      cleaned_env['PATH'] = '/sbin:/bin:/usr/sbin:/usr/bin'
```

Comment:
In our case I think we should add the puppetlabs bin path to it, like we do here:

https://github.com/theforeman/kafo_parsers/blob/2c76581b9e9175701d0a953885bb4e84b5c94e33/lib/kafo_parsers/puppet_strings_module_parser.rb#L59